packages/google_maps_flutter/google_maps_flutter_ios/lib/src/google_maps_flutter_ios.dart

```diff
@@ -477,6 +500,7 @@ class GoogleMapsFlutterIOS extends GoogleMapsFlutterPlatform {
     Set<Polyline> polylines = const <Polyline>{},
     Set<Circle> circles = const <Circle>{},
     Set<TileOverlay> tileOverlays = const <TileOverlay>{},
+    Set<ClusterManager> clusterManagers = const <ClusterManager>{},
```

Comment:
Oops, I missed this before; this change shouldn't be made. It compiles because it's an optional param, but nothing should ever pass this, so we don't need to add it.

Reply:
I'll remove these...
Noticed that this is for slipped through for android, and should be reverted.
https://github.com/flutter/packages/blob/4354f65e1314755f13d4d9db28b9fc08096b8f81/packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart#L601

Reply:
I figured it probably did when I realized I'd missed it here; it's not a big deal but it wouldn't hurt to clean it up later.

Reply:
Actually, scratch that; it would be a breaking change to the Android implementation package to remove it. That's not worth the churn.

Reply:
These lines are now removed